doc/pipeline.rst

```diff
@@ -354,25 +349,23 @@ then creates the DTM using the :ref:`writers.gdal`.
       "pipeline":[
           "autzen-full.las",
           {
-              "type":"ground",
-              "approximate":true,
+              "type":"filters.pmf",
```

Comment:
we should change this whole block to `filters.smrf`.  We shouldn't encourage `filters.pmf` at all